doc/users/next_whats_new/pcolormesh_alpha_improvement.rst

```diff
@@ -0,0 +1,40 @@
+pcolormesh has improved transparency handling by enabling sub-pixel snapping
+----------------------------------------------------------------------------
+
+Due to how the snapping keyword argument was getting passed to the AGG backend,
+previous versions of Matplotlib would appear to show lines between the grid
+edges of a mesh with transparency. This version now applies sub-pixel snapping
```

Comment:
I thought this was snapping to full pixels?  (Maybe I'm just confused.)

Reply:
Good question, I'm confused now too!... My recollection from reading the AGG documentation a while ago was that it has sub-pixel resolution (I think I recall there was an extra buffer added somewhere for this in the scanlines or similar?). I may have conflated the sub-pixel resolution with the ability to snap at a sub-pixel level.

Here is a write-up on the text pixel positioning (sub-pixel level) http://agg.sourceforge.net/antigrain.com/research/font_rasterization/
I'm not entirely clear if that is applicable to these other geometries though.

Finding documentation on all of this is quite challenging, so if you are pretty sure it is actually to pixels then I can just drop the 'sub'.

Reply:
The quick and dirty solution would be to just write "snapping" without saying snapping to what ;) TBH 1) I'm not super certain here either, I think it's pixel snapping but would need to dig more into it (which I'm not going to do right now) and 2) I don't think that many users will care about the technical details anyways ;)

Reply:
Good suggestion :) I just removed 'sub-pixel' altogether.